### ISSUE TITLE
Feature Request: Make window movable and resizeable

### ISSUE THREAD

**Issue by Daniel-Leinberger** [dillonkearns/mobster#80]
First of all, I really like to use this tool! It's really awesome and a really nice product and I want to thank you to developing this. But there's one thing, really bothering me:

The window Popup, when the time runs out. Sure it's helpfull so you can see that the time is over, but we had it a couple of times now, that someone was just programming, while the window pops up. We would highly appreciate it, if there was just a small window or something, so we can finish the line of code and don't get interrupted that much. Like we can resize the window to our own preferences. Therefore, when the time runs out and the window shows up, we could just click away to our main window and finish the line of code.

It would be also very helpfull, if we could move the window around, so the fullsized window, pop ups on a second screen or something and don't get targeted. That would really helps the flow of coding, because you can see that the time runs out and remind everyone, but you don't have to click again somewhere to finish the task.

**_TLDR:
I would appreciate if the programm gets the following features:_**

- [ ] Window becomes resizable (in all directions)

- [ ]  Window becomes movable

- [ ]  Window won't be force targeted on popup (you don't have to click 'continue' to finish a short task)

- [ ] Window pops up on the screen I moved it to
